src/video/SDL_clipboard.c

```diff
@@ -22,6 +22,23 @@
 
 #include "SDL_sysvideo.h"
 
+int
+SDL_SetClipboardData(SDL_ClipboardDataCallback callback, size_t mime_count, const char **mime_types, void *userdata)
+{
+    SDL_VideoDevice *_this = SDL_GetVideoDevice();
+    int status = 0;
+
+    if (_this == NULL) {
+        return SDL_SetError("Video subsystem must be initialized to set clipboard text");
+    }
+
+    if (_this->SetClipboardData) {
+        status = _this->SetClipboardData(_this, callback, mime_count, mime_types, userdata);
+    }
```

Comment:
Suggest returning SDL_Unsupported() if _this->SetClipboardData isn't provided?
Alternatively, the clipboard data can be managed internally so at least the application can provide copy/paste of multiple mime-types to itself.

Reply:
I'll go with unsupported in this PR. I'll take a look at potentially handling it internally as I'm implementing X11.